File: app/src/main/java/com/example/nilvabtchallenge/MainActivity.java
package com.example.nilvabtchallenge;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.NavController;
import androidx.navigation.Navigation;

import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    private NavController navController;
    private View bluetoothNavHost;


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        navController = Navigation.findNavController(MainActivity.this, R.id.navHostBluetoothFragment);
        bluetoothNavHost = findViewById(R.id.navHostBluetoothFragment);

    }
}

File: app/src/main/java/com/nikmaram/bluetoothchat/ui/MainActivity.kt
package com.nikmaram.bluetoothchat.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Parcel
import android.os.Parcelable
import androidx.databinding.DataBindingUtil
import com.nikmaram.bluetoothchat.R
import com.nikmaram.bluetoothchat.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    companion object{
        lateinit var bindingActivity: ActivityMainBinding
    }


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        bindingActivity = DataBindingUtil.setContentView(this, R.layout.activity_main);
    }


}